srcpkgs/recoll/template

```diff
@@ -1,18 +1,21 @@
 # Template file for 'recoll'
 pkgname=recoll
-version=1.30.0
+version=1.32.7
 revision=1
 build_style=gnu-configure
 build_helper=qmake
-configure_args="--disable-python-module --disable-python-chm
- --enable-recollq --disable-x11mon"
-hostmakedepends="automake libtool gettext-devel qt5-qmake pkg-config
- qt5-host-tools"
-makedepends="xapian-core-devel zlib-devel libuuid-devel qt5-webkit-devel qt5-devel
- aspell-devel libxslt-devel libxml2-devel"
+configure_args="--enable-recollq --disable-python-chm --disable-x11mon
+ --disable-webkit --without-systemd"
```

Comment:
does `--without-systemd` change anything in the program logic other than just not installing systemd unit files? If it's just unit files, let's leave them there because this package doesn't have a runit service already

Reply:
it is implicitly enabled by the confs (they are looking for systemd paths/files); so I just added it explicitly (in case something changes)

Reply:
does it do anything other than just installing the systemd service when enabled, though?

from manual.md:

> If the package installs a systemd service file or other unit, leave it in place as a reference point so long as including it has no negative side effects.
> 
> Examples of when not to install systemd units:
> 
> 1. When doing so changes runtime behavior of the packaged software.
> 2. When it is done via a compile time flag that also changes build dependencies.

Reply:
from the documentation 
```
--without-systemd 
   Disable the automatic installation of systemd unit files. 
   Normally unit files are installed if the install path can be detected.
```

It only installs services, BUT
even if removed, it won't install any extra files or services - thus adding the flag to make it explicit.

(As can be seen by the current in repo pkg (without flag), there aren't any systemd unit files installed)

Reply:
ok sounds good